backend/src/main/java/ai/verta/modeldb/experimentRun/ExperimentRunServiceImpl.java

```diff
@@ -199,13 +202,16 @@ public void createExperimentRun(
       experimentRun = experimentRunDAO.insertExperimentRun(projectDAO, experimentRun, userInfo);
       CreateExperimentRun.Response response =
           CreateExperimentRun.Response.newBuilder().setExperimentRun(experimentRun).build();
+      GetResourcesResponseItem entityResource =
+          roleService.getEntityResource(
+              experimentRun.getProjectId(), ModelDBServiceResourceTypes.PROJECT);
       saveAuditLogs(
           Optional.of(userInfo),
           ModelDBServiceActions.CREATE,
           Collections.singletonList(experimentRun.getId()),
           ModelDBUtils.getStringFromProtoObject(request),
           ModelDBUtils.getStringFromProtoObject(response),
-          null);
+          entityResource.getWorkspaceId());
```

Comment:
`getWorkspaceServiceId`, right? Otherwise I think it will get the ID from before workspace service was a thing

Reply:
entityResource means `GetResourcesResponseItem` which is contains only `getWorkspaceId`

Reply:
Oh, my bad =)